src/RPackage-Core/RPackage.class.st

```diff
@@ -1045,21 +1045,16 @@ RPackage >> register [
 { #category : #'private - register' }
 RPackage >> registerClass: aClass [
 	"Private method that declares the mapping between a class and its package."
-	self organizer
-		registerPackage: self forClass: aClass
-]
 
-{ #category : #'private - register' }
-RPackage >> registerClassName: aClassNameSymbol [
-	"Private method that declares the mapping between a class and its package."
-	self organizer
-		registerPackage: self forClassName: aClassNameSymbol
+	self organizer registerPackage: self forClass: aClass
 ]
 
 { #category : #'private - register' }
-RPackage >> removeAllMethodsFromClassNamed: aClassNameSymbol [
-	"Remove all the methods (defined and extensions) that are related to class aClassNameSymbol. aClassNameSymbol is the name of a class and not a metaclass - see class comment."
+RPackage >> removeAllMethodsFromClass: aClass [
+	"Remove all the methods (defined and extensions) that are related to the class as parameter. The class should always be instance side."
 
+	| aClassNameSymbol |
+	aClassNameSymbol := aClass originalName asSymbol.
```

Comment:
Here we can do `aClass originalName asSymbol -> aClass instanceSide originalName asSymbol.`

Reply:
But it was true for the named version but I do not think this is correct for the class version because the metaclass has methods too. So ignore it.